src/engraving/types/propertyvalue.h

```diff
@@ -396,7 +396,7 @@ class PropertyValue
 #endif
         }
 
-        assert(at);
+        //assert(at);
```

Comment:
Seems this assert can really go, as the case it gets hit is being delt with

Reply:
Then again it is hit exactly when setting the font face for staff text, which seems to be where the vtests fail.